src/excmds.ts

```diff
@@ -128,6 +128,55 @@ export async function getNativeVersion(): Promise<void> {
     Native.getNativeMessengerVersion()
 }
 
+/** @hidden
+ * This function is used by rssexec and rssexec completions.
+ */
+//#content
+export async function getRssLinks(): Promise<{ type: string; url: string; title: string }[]> {
+    let seen = new Set<string>()
+    return (Array.from(document.querySelectorAll("link[rel='alternate']")) as HTMLLinkElement[]).filter(e => e.type && e.href && (e.type.indexOf("rss") >= 0 || e.type.indexOf("atom") >= 0) && !seen.has(e.href) && seen.add(e.href)).map(e => {
```

Comment:
I think it might be better to look for links that look like RSS throughout the page. E.g, on this page https://www.bbc.co.uk/news/10628494 I was expecting to be able to choose from all of the links.